app/actions.ts

```diff
@@ -0,0 +1,53 @@
+'use server';
+import prisma from '@/lib/prisma';
+
+const slugify = (text: string) =>
+  text
+    .toString()
+    .normalize('NFD')
+    .replace(/[\u0300-\u036f]/g, '')
+    .toLowerCase()
+    .trim()
+    .replace(/\s+/g, '-')
+    .replace(/[^\w-]+/g, '')
+    .replace(/--+/g, '-');
+
+export const createBusiness = async (
```

Comment:
Love this! Is this run on the server then? I need to look into what dictates that.